misc/sampleconfigs/gpio-buttons.py.sample

```diff
@@ -127,9 +128,9 @@ GPIO.setup(btn_volD       , GPIO.IN, pull_up_down=GPIO.PUD_UP)
 GPIO.setup(btn_next       , GPIO.IN, pull_up_down=GPIO.PUD_UP)
 GPIO.setup(btn_prev       , GPIO.IN, pull_up_down=GPIO.PUD_UP)
 GPIO.setup(btn_halt       , GPIO.IN, pull_up_down=GPIO.PUD_UP)
-#GPIO.setup(btn_rfidoff, GPIO.IN, pull_up_down=GPIO.PUD_UP)
+#GPIO.setup(btn_rfidoff    , GPIO.IN, pull_up_down=GPIO.PUD_DOWN)
```

Comment:
Not sure, why it’s changed from up to down?

Reply:
Cause the action should First be triggered after the gpio is pulled down or RFID chip ist removed.